### PR TITLE
Puts val loss on cpu for np.mean()

### PR DESCRIPTION
**Pull request by Adamits**
I guess I had not tested the final evaluators update on gpu. Since we now get the mean val loss with numpy.mean, we need to be sure the loss tensors are on cpu.

### CODE REVIEW

**Comment by kylebgorman** [comment]
What if we just use PyTorch's mean? Does that solve the problem? 

Two interfaces for it:

https://pytorch.org/docs/stable/generated/torch.mean.html
https://pytorch.org/docs/stable/generated/torch.Tensor.mean.html

**Comment by Adamits** [comment]
> What if we just use PyTorch's mean? Does that solve the problem?
> 
> Two interfaces for it:
> 
> https://pytorch.org/docs/stable/generated/torch.mean.html https://pytorch.org/docs/stable/generated/torch.Tensor.mean.html

Good point, thats a better solution.

**Comment by Adamits** [comment]
Actually, the issue is that we end up with a list, not a tensor, and I think the overhead of turning the list into a tensor first is probably > the overhead of moving the values to CPU. I can test though.

**Comment by kylebgorman** [comment]
> Actually, the issue is that we end up with a list, not a tensor, and I think the overhead of turning the list into a tensor first is probably > the overhead of moving the values to CPU. I can test though.

yeah whatever's faster (or if they're the same pick the one that's more elegant...)

**Review by kylebgorman — APPROVED** [review]
LGTM.

**Comment by kylebgorman** [comment]
Shall I merge?

**Comment by Adamits** [comment]
Yes!
